crates/astria-proto/proto/astria/sequencer/v1alpha2/celestia.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+
+package astria.sequencer.v1alpha2;
+
+import "tendermint/types/types.proto";
+import "astria/sequencer/v1alpha2/validation.proto";
+
+// An "item blob" of rollup transactions submitted to celestia.
+//
+// The transactions contained in the item belong to a rollup identified
+// by `chain_id`, and were included in the sequencer block identified
+// by `sequencer_block_hash`.
+message CelestiaItem {
```

Comment:
does this correspond to the `RollupNamespaceData` type? let's make the names match up

Reply:
It does, but I want to move away from relatively generic sounding names like `RollupNamespaceData` and use celestia specific naming instead (because these are at the end of the day only ending up on celestia).

I am not a fan of `CelestiaItem`, but I would like something more specific than `RollupNamespaceData`

Reply:
true, in my head `(Something)NamespaceData` is celestia-specific since namespaces are for celestia, but I understand the reasoning. maybe `CelestiaRollupData` and `CelestiaSequencerData` or something?

Reply:
replacing 'Celestia' with 'DataAvailability' in naming would adhere to the design of keeping the choice of data availability provider open to the user

Reply:
@noot I renamed the "items" to `CelestiaRollupData` and the head to `CelestiaSequencerData`.

@emhane Yes, the design vision is to let users choose their preferred rollup. Unfortunately we still need to define rollup-specific datastructures to interact with them. The goal of this PR is to make this clear.